core/src/main/kotlin/org/evomaster/core/search/service/AverageCalculator.kt

```diff
@@ -0,0 +1,27 @@
+package org.evomaster.core.search.service
+
+class AverageCalculator {
```

Comment:
why this class?  wouldn't `org.evomaster.core.utils.IncrementalAverage` handle this?
also, as this is not an injectable service, it should not be under a `*.service` package.

Reply:
I didn't know about this class. I will replace it with IncrementalAverage.